src/views/ApartmentApp/index.js

```diff
@@ -2,13 +2,15 @@
 import React, { Component } from 'react';
 import 'date-fns';
 import { Grid, Card, CardHeader, CardContent, Button, Typography } from '@material-ui/core/';
-// import GordonDialogBox from '../../components/GordonDialogBox';
 import GordonLoader from '../../components/Loader';
+import AlertDialogBox from '../../components/AlertDialogBox';
 import SimpleSnackbar from '../../components/Snackbar';
 import ApplicantList from '../../components/ApartmentApplicantList';
 import user from '../../services/user';
 import housing from '../../services/housing';
 import './apartmentApp.css';
+const MAX_NUM_APPLICANTS = 8;
+// const MIN_NUM_APPLICANTS = 2;
```

Comment:
Should you just delete the MIN?

Reply:
Probably. If so, I will do so in apartapp-landing-page #911  because most of this student-related code was moved from `.../ApartmentApp/index.js` to `.../ApartmentApp/components/StudentApplication/index.js`

Reply:
Okay, sounds good